core/lens/lensHelpers.ml

```diff
@@ -13,13 +13,7 @@ let query_count = ref 0
 let print message =
   (if false then print_endline message)
 
-let ensure_lenses_enabled () =
-  if Settings.get_value Basicsettings.RelationalLenses.relational_lenses then
-    ()
-  else
-    failwith "Code uses relational lenses, but relational lenses are not enabled. Please set the relational lenses flag."
-
-module Lens = struct
+module Lens' = struct
```

Comment:
Is it a good idea to have primes in a module name?  (Or is there some reason it's bad to ahve "Lens.Helpers.Lens"?  

I tend to agree that "Lens" is not a very descriptive name for this submodule, anyway.

Reply:
I agree that the module `Lens'` should have another name, but I will let @rudihorn decide on the API. I don't like the primed name either, but it needs some name that is different from `Lens` to avoid a clash with the encompassing module `Lens`.